src/components/Index/BillContent.svelte

```diff
@@ -15,13 +15,53 @@
 		bills: Pick<Bill, 'id' | 'title' | 'nickname'>[];
 	}
 
+	type MpTerm = {
+		name: string;
```

Comment:
You use `name` as a key here. I suggest you use `id` instead since it guarantee to be unique from the database to GraphQL query.